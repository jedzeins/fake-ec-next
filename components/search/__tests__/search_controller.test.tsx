import React from 'react';
import renderer from 'react-test-renderer';
import SearchController from '../searchController';


describe('Search controller', ()=> {

    test('Tests Search Controller',()=> {
        const el = renderer.create(<SearchController/>);
        let tree = el.toJSON();
        expect(tree).toMatchSnapshot();
    })
})