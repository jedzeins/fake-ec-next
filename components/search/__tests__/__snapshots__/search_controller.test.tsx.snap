// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search controller Tests Search Controller 1`] = `
<article
  className="search"
>
  <input
    className="input false"
    onBlur={[Function]}
    onChange={[Function]}
    type="text"
    value=""
  />
  <button
    className="button"
  >
    <svg
      color="purple"
      fill="currentColor"
      height="20px"
      stroke="currentColor"
      strokeWidth="0"
      style={
        Object {
          "color": "purple",
        }
      }
      viewBox="0 0 16 16"
      width="20px"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M15.7 13.3l-3.81-3.83A5.93 5.93 0 0 0 13 6c0-3.31-2.69-6-6-6S1 2.69 1 6s2.69 6 6 6c1.3 0 2.48-.41 3.47-1.11l3.83 3.81c.19.2.45.3.7.3.25 0 .52-.09.7-.3a.996.996 0 0 0 0-1.41v.01zM7 10.7c-2.59 0-4.7-2.11-4.7-4.7 0-2.59 2.11-4.7 4.7-4.7 2.59 0 4.7 2.11 4.7 4.7 0 2.59-2.11 4.7-4.7 4.7z"
        fillRule="evenodd"
      />
    </svg>
  </button>
</article>
`;
